tests/testHavenLoader.py

```diff
@@ -0,0 +1,23 @@
+import os
+import bpy
+os.environ["INSIDE_OF_THE_INTERNAL_BLENDER_PYTHON_ENVIRONMENT"] = "1"
+import blenderproc as bproc
+
+# This file attempts to load all downloaded Haven textures.
+# Not a real test yet, just what I'm using for now.
+
+home = os.path.expanduser("~")
+haven_folder = os.path.join(home, "assets", "haven")
+haven_textures_folder = os.path.join(haven_folder, "textures")
```

Comment:
This should also be an environment variable. As our automatized tests, else would need have to use the same folder as you did.

Reply:
I couldn't find an example in `tests` of an environment variable being used, so I just tried something. Let me know whether you like it.